cmd/crank/beta/beta.go

```diff
@@ -21,22 +21,18 @@ package beta
 
 import (
 	"github.com/crossplane/crossplane/cmd/crank/beta/convert"
-	"github.com/crossplane/crossplane/cmd/crank/beta/render"
 	"github.com/crossplane/crossplane/cmd/crank/beta/top"
 	"github.com/crossplane/crossplane/cmd/crank/beta/trace"
 	"github.com/crossplane/crossplane/cmd/crank/beta/validate"
-	"github.com/crossplane/crossplane/cmd/crank/beta/xpkg"
 )
 
 // Cmd contains beta commands.
 type Cmd struct {
 	// Subcommands and flags will appear in the CLI help output in the same
 	// order they're specified here. Keep them in alphabetical order.
 	Convert  convert.Cmd  `cmd:"" help:"Convert a Crossplane resource to a newer version or kind."`
-	Render   render.Cmd   `cmd:"" help:"Render a composite resource (XR)."`
```

Comment:
If we wanted to avoid a breaking change we could import the non-beta commands here.

I'm not sure how important that is though - presumably we'll want to remove them from under `beta` at some point?